src/core/qgsapplication.h

```diff
@@ -465,6 +471,7 @@ class CORE_EXPORT QgsApplication : public QApplication
     QgsActionScopeRegistry* mActionScopeRegistry;
     QgsRuntimeProfiler* mProfiler;
     QgsTaskManager* mTaskManager;
+    QgsFieldFormatterRegistry* mFieldFormatterRegistry;
```

Comment:
Shouldn't all those pointers be replaced by `QScopedPointer`s? Or, even better, values?